app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+from datetime import datetime
+
+import requests
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL, {"key": API_KEY, "q": FILTERING})
+    data = response.json()
+
+    if response.status_code == 200:
+        city = data["location"]["name"]
+        region = data["location"]["region"]
+        localtime = datetime.strptime(
+            data["location"]["localtime"], "%Y-%m-%d %H:%M"
```

Comment:
Do we need "%Y-%m-%d %H:%M"? API gives us formatted data

Reply:
so why did you approve? @rostyak

Reply:
rostyak is right, you can leave just `data["location"]["localtime"]`